UKSI/scripts/20 - match to existing taxa.sql

```diff
@@ -15,7 +15,7 @@ SET id=t.id,
     OR COALESCE(pt.taxon_rank_id, 0)<>COALESCE(t.taxon_rank_id, 0)
     OR COALESCE(pt.attribute, '')<>COALESCE(t.attribute, '')
     OR pt.marine_flag<>t.marine_flag
-  )
+  ) OR {{ force_cache_rebuild }} 
```

Comment:
I wonder whether, rather than not setting the changed flag here, it would be better to set the flag anyway (as logically the records have changed), it would be better to check the state of this flag later when deciding whether to do the cache update?

Reply:
I'll revisit.

Reply:
I left this in place for now John as I was unclear on what would be affected if I moved it elsewhere. However, I haven't had need to use it again.